ui/components/multichain/connect-accounts-modal/connect-accounts-modal-list.tsx

```diff
@@ -28,6 +28,7 @@ import { getURLHost } from '../../../helpers/utils/util';
 import { addPermittedAccounts } from '../../../store/actions';
 import { ConnectAccountsListProps } from './connect-account-modal.types';
 
+// check usage of this component
```

Comment:
TODO

Reply:
used on the ConnectionsPage. still need to verify deeper

Reply:
looking at this

Reply:
Seeing it in `routes.component.js` but unsure if this route is ever used now?

Asking @darkwing : https://consensys.slack.com/archives/C06FXU326RL/p1743616755737269